src/css/wazimap-ng-v1.webflow.css

```diff
@@ -2032,6 +2032,7 @@ label {
   -webkit-align-items: center;
   -ms-flex-align: center;
   align-items: center;
+  z-index: 999;
```

Comment:
can you add the `z-index` to the `setAttribute` otherwise this needs a change from matt. we might want to change that in the future, but for now this is the fastest.

Reply:
Added